<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>172</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Not idle task</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>561</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Application settings</string>
    </property>
    <widget class="QLabel" name="LABEL_IDLE_INTERVAL">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Idle Interval: (in seconds):</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="EDIT_PATH">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="LABEL_START_PATH">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Application to start:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CHECKBOX_KILL_APP_IF_EXIT">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>When exiting kill application</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="EDIT_INTERVAL">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="BUTTON_OK">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>105</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_CANCEL">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>105</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
